pkg/virtualization/core/host/hostcomputersystem.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/microsoft/wmi/pkg/virtualization/network/ethernetport"
 	"github.com/microsoft/wmi/pkg/virtualization/network/switchextension"
 	"github.com/microsoft/wmi/pkg/virtualization/network/switchport"
-	"github.com/microsoft/wmi/server2019/root/virtualization/v2"
+	v2 "github.com/microsoft/wmi/server2019/root/virtualization/v2"
```

Comment:
[nitpick] [nitpick] I am just curious about this change. Isn't the new code going to produce the same result?

Reply:
i think go fmt may have done this, i did not pay attention. i will check if that's the case or not.